src/features/search/components/SearchBox/SearchBox.native.test.tsx

```diff
@@ -164,15 +164,6 @@ describe('SearchBox component', () => {
     expect(searchInput).toBeOnTheScreen()
   })
 
-  it('should render SearchBox when FF is enabled', async () => {
-    useFeatureFlagSpy.mockReturnValueOnce(true)
-    renderSearchBox()
-
-    await act(async () => {})
-
-    expect(screen).toMatchSnapshot()
-  })
-
```

Comment:
Il faudrait remplacer ce test pas un teste qui vérifie qu'il y a bien la checkbox soit par un élément qui permet de l'identifier, soit par l'ajout d'un testID (en cas de nom présence d'un élément qui permet d'identifier la checkbox).

Reply:
J'arrive pas à voir de quel checkbox tu parles ?

Reply:
SearchBox pardon !

Reply:
Ah yes je capte mieux ^^

Reply:
La SearchBox est rendue avec ou sans FF du coup le test `should render SearchBox` suffit